code/modules/modular_computers/computers/item/disks/virus_disk.dm

```diff
@@ -143,7 +143,8 @@
 				target_mind = user.mind
 			else
 				target_mind = pick(backup_players)
-		hidden_uplink = target.AddComponent(/datum/component/uplink, target_mind, enabled = TRUE, starting_tc = telecrystals, has_progression = TRUE)
+		hidden_uplink = target.AddComponent(/datum/component/uplink, target_mind, enabled = TRUE, starting_tc = telecrystals, has_progression = TRAIT_CAN_USE_FLIGHT_POTION)
```

Comment:
what's with ``TRAIT_CAN_USE_FLIGHT_POTION``?

Reply:
I have no idea what happened there.

Reply:
Thank you for alerting me, I have fixed it.